optax/_src/transform.py

```diff
@@ -717,12 +717,13 @@ def update_fn(updates, state, params=None):  # pylint: disable=missing-docstring
     treedef = jax.tree_structure(updates)
     count_inc = numerics.safe_int32_increment(state.count)
     variance = eta / count_inc**gamma
+    std = jnp.sqrt(variance)
```

Comment:
Nit: std is a standard abbreviation so it's perfectly fine as is, but I'd personally prefer `standard_deviation`, also to be consistent with the use of `variance` over `var`.

Reply:
Done.
(I chose std to match its use in stochastic_gradient_estimators, but totally agree with your argument)